source/gent/tier1_hortense.rst

```diff
@@ -64,24 +64,49 @@ Getting access
 
 See https://www.vscentrum.be/compute for more information on requesting access.
 
-To access the Tier-1 Hortense cluster, use SSH to connect to the dedicated login node with your VSC account:
+To access the Tier-1 Hortense cluster use the `web portal <https://tier1.hpc.ugent.be/>`_,
+or use SSH to connect to the dedicated login node with your VSC account:
 
 * from the public internet, use ``tier1.hpc.ugent.be``
 * from within the VSC network, use ``tier1.gent.vsc``
 
+You can find more information about the usage of the web portal in chapter "8 Using the HPC-UGent web portal"
+in the `HPC Ugent tutorials <https://www.ugent.be/hpc/en/support/documentation.htm>`_
+
 More general information about SSH login is available at :ref:`acccess_data_transfer`.
 
 .. note::
-  There is currently only one small login node (16 cores, 64GB RAM) available.
-
-  **Please only use the login node as an access portal!**
-
-  For resource-intensive tasks, like software compilation, testing job scripts, etc., please use an interactive job.
+  The ssh key of the login nodes changed. If you get a message like this (or similar):
```

Comment:
in linux (and perhaps macos) we can add the commands to clean this up using ssh-keygen?

Reply:
That belongs in the email to the Tier-1 pilot users, not in the documentation (host keys of login nodes shouldn't change)